MockKit/MockPumpManager.swift

```diff
@@ -15,19 +15,23 @@ public protocol MockPumpManagerStateObserver {
     func mockPumpManager(_ manager: MockPumpManager, didUpdate status: PumpManagerStatus, oldStatus: PumpManagerStatus)
 }
 
-private enum MockPumpManagerError: LocalizedError {
+public enum MockPumpManagerError: LocalizedError {
     case pumpSuspended
     case communicationFailure
     case bolusInProgress
+    case missingSettings
+    
 
-    var failureReason: String? {
+    public var failureReason: String? {
         switch self {
         case .pumpSuspended:
             return "Pump is suspended"
         case .communicationFailure:
             return "Unable to communicate with pump"
         case .bolusInProgress:
             return "Bolus in progress"
+        case .missingSettings:
+            return "Missing Settings."
```

Comment:
[nit] Might want to remove the trailing `.` to be consistent with the other failure reasons here.